src/index.html

```diff
@@ -1,5 +1,8 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
```

Comment:
The `<html>` tag should not be self-closing. It should have a separate closing tag `</html>` at the end of the document.